lego/apps/companies/constants.py

```diff
@@ -83,6 +83,62 @@
     COLLABORATION_REVUE_ANNIVERSARY: "Abakusrevy jubileum",
 }
 
+COMPANY_TYPES_SMALL_CONSULTANT = "company_types_small_consultant"
+COMPANY_TYPES_MEDIUM_CONSULTANT = "company_types_medium_consultant"
+COMPANY_TYPES_LARGE_CONSULTANT = "company_types_large_consultant"
+COMPANY_TYPES_INHOUSE = "company_types_inhouse"
+COMPANY_TYPES_OTHERS = "company_types_others"
+COMPANY_TYPES_START_UP = "company_types_start_up"
+COMPANY_TYPES_GOVERNMENTAL = "company_types_governmental"
+
+COMPANY_TYPES = (
+    (COMPANY_TYPES_SMALL_CONSULTANT, COMPANY_TYPES_SMALL_CONSULTANT),
+    (COMPANY_TYPES_MEDIUM_CONSULTANT, COMPANY_TYPES_MEDIUM_CONSULTANT),
+    (COMPANY_TYPES_LARGE_CONSULTANT, COMPANY_TYPES_LARGE_CONSULTANT),
+    (COMPANY_TYPES_INHOUSE, COMPANY_TYPES_INHOUSE),
+    (COMPANY_TYPES_OTHERS, COMPANY_TYPES_OTHERS),
+    (COMPANY_TYPES_START_UP, COMPANY_TYPES_START_UP),
+    (COMPANY_TYPES_GOVERNMENTAL, COMPANY_TYPES_GOVERNMENTAL),
+)
```

Comment:
I agree with Ivar on making enums. Espescially stuff like this (throughout all of `constants.py`) if possible

Reply:
something like `tuple((e.value, e.value) for e in E_Company_Types)`